p2p/src/authenticated/discovery/types.rs

```diff
@@ -342,4 +443,172 @@ mod tests {
             message_len + MAX_PAYLOAD_DATA_OVERHEAD
         );
     }
+
+    #[test]
+    fn peer_info_verifier_accepts_valid_peer() {
```

Comment:
These were deeply intertwined with tracker previously. They are now much easier to follow 👍